src/SIL.XForge.Scripture/ClientApp/src/app/translate/draft-generation/draft-apply-dialog/draft-apply-dialog.component.html

```diff
@@ -3,53 +3,51 @@
 }
 
 <ng-container *transloco="let t; read: 'draft_apply_dialog'">
+  <h2 mat-dialog-title>{{ t("select_alternate_project") }}</h2>
   <div mat-dialog-content>
     <form [formGroup]="addToProjectForm">
-      <p>{{ t("select_alternate_project") }}</p>
       <app-project-select
         [projects]="projects"
         [placeholder]="t('choose_project')"
         [isDisabled]="projects.length === 0"
-        (projectSelect)="projectSelectedAsync($event.paratextId)"
+        [invalidMessageMapper]="invalidMessageMapper"
+        (projectSelect)="projectSelected($event.paratextId)"
         formControlName="targetParatextId"
       ></app-project-select>
-      @if (!canEditProject && isAppOnline) {
-        <mat-error class="cannot-edit-message mat-hint">{{ t("no_write_permission") }}</mat-error>
-      }
-      @if (projectLoadingFailed && isAppOnline) {
-        <mat-error class="fetch-projects-failed-message">{{ t("error_fetching_projects") }}</mat-error>
-      }
       @if (!isAppOnline) {
         <mat-error class="offline-message">{{ t("connect_to_the_internet") }}</mat-error>
       }
-      @if (connectOtherProject != null) {
-        <div class="unlisted-project-message">
-          <p>
-            <span [innerHTML]="connectOtherProject.before"></span>
-            <a [appRouterLink]="['projects']" (click)="close()">{{ connectOtherProject.templateTagText }}</a>
-            <span>{{ connectOtherProject.after }}</span>
-          </p>
-        </div>
-      }
+      <div class="unlisted-project-message">
+        <a
+          [appRouterLink]="['projects']"
+          (click)="close()"
+          [innerHtml]="i18n.translateAndInsertTags('draft_apply_dialog.looking_for_unlisted_project')"
```

Comment:
@RaymondLuong3 
I have some concern here.
- I really, *really* do not like the use of `innerHtml`, and have been trying hard to avoid it. Angular sanitizes it, but it's still kind of ugly from a security standpoint. 
- It seems odd for the whole sentence to be a link.
- This string has been translated into three languages, so it will have to be re-translated, and only because of a change to the variable name.